src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -878,6 +879,15 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         false, false, "caveman", 50, new PotionEffect(PotionEffectType.FAST_DIGGING, 800, 2))
         .register(plugin);
 
+        new Talisman(SlimefunItems.TALISMAN_WISE,
+        new ItemStack[] { SlimefunItems.MAGIC_LUMP_3, null, SlimefunItems.MAGIC_LUMP_3, SlimefunItems.MAGICAL_GLASS, SlimefunItems.TALISMAN_MAGICIAN, SlimefunItems.FILLED_FLASK_OF_KNOWLEDGE, SlimefunItems.MAGIC_LUMP_3, null, SlimefunItems.MAGIC_LUMP_3},
+        false, false, "wise", 20)
+        .register(plugin);
+
+        new ResurrectedTalisman(SlimefunItems.TALISMAN_RESURRECTED,
+        new ItemStack[] { SlimefunItems.MAGIC_LUMP_3, null, SlimefunItems.MAGIC_LUMP_3, SlimefunItems.ESSENCE_OF_AFTERLIFE, SlimefunItems.TALISMAN_ANGEL, SlimefunItems.ELYTRA_SCALE, SlimefunItems.MAGIC_LUMP_3, null, SlimefunItems.MAGIC_LUMP_3 })
```

Comment:
One Essence of afterlife in the middle top and two elytra scales on the sides in the middle?

Reply:
Sure, the Essense of afterlife is the hardest peace to craft here so I dont think itll matter too much, but its a little exta effort they need to put in for this item